Podpis i GnuPG/sprawozdanie podpis.md

```diff
@@ -0,0 +1,108 @@
+# Podpis i GnuPG
+
+Członkowie zespołu:
+- Jarosław Dakowicz
+- Piotr Kozioł
+- Anton Maisiuk
+
+## 1. System podpisu elektronicznego
+### 1.1 Przygotowywanie systemu
+
+Za pomocą bibliotek `node-rsa` i `crc` został stworzyny system do podpisywania elektronicznego
```

Comment:
Najprawdopodobniej chodziło o to, żeby te algorytmy samemu zaimplementować, a nie korzystać z zewnętrznych bibliotek. No ale może Grześ tego nawet nie zauważy...